### PR TITLE
www-plugins/browserpass: new package

### PR DESCRIPTION
**Pull request by lluixhi**
The Browserpass package provides the host binary necessary for the Firefox or Chrome browser extension to read your `app-admin/pass` password store (for logging into websites).

Unlike [passff](https://github.com/passff/passff) which uses the app-admin/pass binary, browserpass only requires app-crypt/gnupg to read the password-store.

I ended up packaging this partially because browser extensions requiring WebExtension host binaries are going to end up getting more common, and I was curious about the golang eclasses.

### CODE REVIEW

File: dev-go/qr/qr-0_p20161121.ebuild [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+else
+	KEYWORDS="~amd64"
+	EGIT_COMMIT=48b2ede4844e13f1a2b7ce4d2529c9af7e359fc5
+	SRC_URI="https://github.com/rsc/qr/archive/${EGIT_COMMIT}.tar.gz -> ${P}.tar.gz"
+	inherit golang-vcs-snapshot
+fi
+inherit golang-build
+
+DESCRIPTION="Basic QR code library for Go"
+HOMEPAGE="https://${EGO_PN}"
```
Comment (mgorny) [inline dev-go/qr/qr-0_p20161121.ebuild]: Nope, variables in `HOMEPAGE` are forbidden.

File: dev-go/qr/qr-0_p20161121.ebuild [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+LICENSE="MIT"
+SLOT="0/${PVR}"
+RDEPEND=""
+
+src_install() {
+	golang-build_src_install
+
+	pushd "src/${EGO_PN}" || die
```
Comment (mgorny) [inline dev-go/qr/qr-0_p20161121.ebuild]: `>/dev/null`

File: dev-go/qr/qr-0_p20161121.ebuild [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+	einstalldocs
+	popd || die
```
Comment (mgorny) [inline dev-go/qr/qr-0_p20161121.ebuild]: Likewise.

File: www-plugins/browserpass/browserpass-2.0.2.ebuild [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+	inherit golang-vcs
+else
+	KEYWORDS="~amd64"
+	EGIT_COMMIT=${PV}
+	SRC_URI="https://${EGO_PN}/archive/${EGIT_COMMIT}.tar.gz -> ${P}.tar.gz"
+	inherit golang-vcs-snapshot
+fi
+inherit golang-build
+
+DESCRIPTION="WebExtension host binary for pass, a UNIX password manager."
```
Comment (mgorny) [inline www-plugins/browserpass/browserpass-2.0.2.ebuild]: This is not a full sentence, so no trailing `.`

**Comment by lluixhi** [comment]
@mgorny 
Fixed the issues you noticed

**Pull request closed without merging**